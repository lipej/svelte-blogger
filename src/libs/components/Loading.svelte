<div class="flex justify-center items-center">
	<button class="btn btn-xl btn-ghost loading text-xl" />
</div>
