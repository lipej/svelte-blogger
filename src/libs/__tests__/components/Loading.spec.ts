import { render } from '@testing-library/svelte';
import Loading from '../../components/Loading.svelte';

describe('Loading component', () => {
	it('should render Loading', () => {
		const { container } = render(Loading);

		const link = container.querySelector('button');

		expect(link.classList.toString()).toMatchSnapshot();
	});
});
