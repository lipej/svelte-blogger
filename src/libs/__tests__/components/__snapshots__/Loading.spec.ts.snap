// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render Loading 1`] = `"btn btn-xl btn-ghost loading text-xl"`;
